packages/blocks-ui/src/transforms.js

```diff
@@ -132,6 +132,7 @@ export const reorderJSXBlocks = (code, drag) => {
 }
 
 export const insertJSXBlock = (code, { blocks, ...drag }) => {
+  console.log('blocks:', blocks)
```

Comment:
```suggestion
```

@andrew-t-james This think was included by accident?

Reply:
@johnletey yup my bad.